Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1141,14 +1141,84 @@ open class NavigationMapView: UIView {
         updateRouteDurations(along: visibleRoutes)
     }
     
+    /**
+     Controls if displayed `continuousAlternative`s will also be annotated with estimated travel time delta relative to the main route.
+     
+     Callouts text font also respects `NavigationMapView.routeDurationAnnotationFontNames` property.
+     Default value is `true`.
+     */
+    public var showsContinuousAlternativeRoutesRelativeDuration: Bool = true {
```

Comment:
Do developers need the ability to show and hide the callouts dynamically? If not, if they only need the ability to decide whether to show or hide them when showing alternative routes, then `show(continuousAlternatives:)` should take a parameter that controls whether `showContinuousAlternativeRoutesDurations()` is called. That way NavigationMapView doesn’t need to maintain extra state. If necessary, NavigationViewController and CarPlayNavigationViewController can have a similar property.

As for `removeContinuousAlternativesRoutes()`, removing alternative routes should always remove their callouts.

Reply:
I think it is more versatile to allow toggling it on and off arbitrarily. I can imagine scenarios where user may hide or show the callouts without redrawing the alternative routes. Moving this property to NavigationViewController seems to create an overhead since it would required to update the route lines by calling `show(continuousAlternatives:)` on each toggle, which does more operations than needed.
`removeContinuousAlternativesRoutes` does also remove the callouts.

Reply:
It’s a bit inconsistent that we have a property controlling callouts on continuous alternative routes but an explicit method for showing callouts on one-shot alternative routes. For one-shot alternative callouts, we avoided the overhead of redrawing the routes themselves by splitting out a public `showRouteDurations(along:)` method as a separate step.